src/renderer/components/ft-shaka-video-player/ft-shaka-video-player.css

```
@@ -2067,12 +2084,90 @@
+  font-weight: 700;
+  line-height: 1;
+  background: #fff;
+  content: attr(data-point);
+  text-shadow: none;
+}
+
+:deep(.abRepeatMarkerB) {
+  color: var(--primary-color);
+}
+
+:deep(.abRepeatMarkerB::after) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":deep" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)